src/sdk.ts

```
@@ -1450,10 +1475,13 @@ export type {
   UpdateAppOptions,
   UpdateChannelOptions,
   UpdateOrganizationOptions,
+  ProbeOptions,
```

ESLint

[error] Expected "ProbeOptions" to come before "UpdateOrganizationOptions".

(perfectionist/sort-named-exports)